packages/graphql/__tests__/node.spec.js

```diff
@@ -0,0 +1,72 @@
+/* eslint-env node, jest */
+
+var React = require('react');
+
+var mockResponse = {
+  __schema: {
+    types: [
+      {
+        kind: 'INTERFACE',
+        name: 'Character',
+        possibleTypes: [{ name: 'Human' }, { name: 'Droid' }],
+      },
+    ],
+  },
+};
+
+describe('graphql node extension', function() {
+  var hopsGraphql;
+  var constants = require('../lib/constants');
+
+  beforeEach(function() {
+    jest.mock('react-apollo', function() {
+      return {
+        getDataFromTree: jest.fn(function() {
+          return Promise.resolve();
+        }),
+      };
+    });
+    jest.mock('../lib/util', function() {
+      return {
+        getIntrospectionResult: jest.fn(function() {
+          return mockResponse;
+        }),
+      };
+    });
+    hopsGraphql = require('../node');
+  });
+
+  it('should set ssrMode', function() {
+    var context = new hopsGraphql.GraphQLContext();
+
+    expect(context.client.ssrMode).toBe(true);
+  });
+
+  it.only('should not prefetch in static mode', function() {
```

Comment:
`.only` needs to be removed.
For mocha we had an eslint rule in place to catch such errors. Maybe we should do the same for jest. :)